published/external/xdpapi.h

```diff
@@ -92,6 +92,22 @@ XDP_GET_ROUTINE_FN(
 //
 #define XDP_API_VERSION_LATEST XDP_API_VERSION_2
 
+inline
+BOOLEAN
+XdpIsApiVersionSupported(
```

Comment:
I'm not entirely sure this should be in a public header - it should be up to each app, and up to the platform, to independently determine what they support